static/components/PoiCard.js

```diff
@@ -33,9 +33,25 @@ class PoiCard extends React.Component {
     return Math.trunc(R * c)
   }
 
+  hoverCard() {
+    this.base.classList.toggle('current-card')
```

Comment:
`toggle` will add if it is not present or remove it otherwise.
Are you sure this is the behavior your want?